neuraxle/steps/misc.py

```diff
@@ -161,7 +160,7 @@ def inverse_transform(self, processed_outputs):
         return processed_outputs
 
 
-class CallbackWrapper(ForceMustHandleMixin, MetaStepMixin, BaseStep):
+class CallbackWrapper(HandleOnlyMixin, MetaStepMixin, BaseStep):
```

Comment:
what is a `CallbackWrapper` ? i dont know what this class does. I might have asked to document it in a previous PR to understand more.